theseus_gui/src/components/ui/Navigation.vue

```diff
@@ -0,0 +1,180 @@
+<script setup>
+import {
+  SunIcon,
+  MoonIcon,
+  SearchIcon,
+  BookIcon,
+  ClientIcon,
+  PlusIcon,
+  SettingsIcon,
+} from 'omorphia'
+import { useStore } from 'vuex'
+import { RouterLink } from 'vue-router'
+import SideNav from '../SideNav.vue'
+import UserSection from './UserSection.vue'
+import Instance from './Instance.vue'
+
+const { state, commit } = useStore()
+
+commit('fetchInstances')
+</script>
+
+<template>
+  <SideNav class="navigation">
+    <UserSection />
+    <div id="pages">
+      <RouterLink
+        to="/"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <ClientIcon />Home</RouterLink
+      >
+      <RouterLink
+        to="/browse"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <SearchIcon />Browse</RouterLink
+      >
+      <RouterLink
+        to="/library"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <BookIcon />Library</RouterLink
+      >
+    </div>
+    <div id="instances">
+      <p>Instances</p>
+      <Instance v-for="instance in state.instances" display="list" :instance="instance" />
+    </div>
```

Comment:
For this section, why not use the omorphia button component over the raw router link?

Reply:
Closing for now after internal discussion.